src/Chainweb/RestAPI/NodeInfo.hs

```diff
@@ -38,7 +44,12 @@ data NodeInfo = NodeInfo
     nodeVersion :: ChainwebVersion
   , nodeApiVersion :: Text
   , nodeChains :: [Text]
+  -- ^ Current list of chains
   , nodeNumberOfChains :: !Int
+  -- ^ Current number of chains
+  , nodeGraphHistory :: [(BlockHeight, [(Int, [Int])])]
```

Comment:
This should only include the *known name* of the graph, not the graph structure. 

The info endpoint is hit potentially very often and it should be very efficient, both in terms of server side compute and in terms of payload size. We expect chainweb graphs to grow very large, so we shouldn't include their (well known static) structure into the response from the info endpoint.

Reply:
As discussed, after this release we will be moving the `info` endpoint out of the `peer` network, to enable putting these kinds of services behind a load balancer etc, and eventually not have to serve this from a node at all potentially.